sdk/keyvault/azure-security-keyvault-certificates/test/ut/certificate_client_test.cpp

```diff
@@ -805,7 +821,7 @@ TEST_F(KeyVaultCertificateClientTest, DownloadImportPem)
     options.Properties.Name = importName;
     auto imported = client.ImportCertificate(importName, options).Value;
 
-    EXPECT_EQ(imported.Properties.Name, importName);
+    //    EXPECT_EQ(imported.Properties.Name, importName);
```

Comment:
Commented out code has no clear course of action. If we want to keep these commented, please log an issue, and insert a comment with a link to that issue before every test line that you are commenting out in this PR.

Reply:
done